pennylane/qchem/openfermion_obs.py

```diff
@@ -978,4 +1020,66 @@ def molecular_hamiltonian(
 
     h_pl = qml.qchem.convert.import_operator(h_of, wires=wires, tol=convert_tol)
 
-    return h_pl, qubits
+    return h_pl, len(h_pl.wires)
+
+
+def _pyscf_integrals(
+    symbols,
+    coordinates,
+    charge=0,
+    mult=1,
+    basis="sto-3g",
+    active_electrons=None,
+    active_orbitals=None,
+):
+    r"""Compute pyscf integrals."""
+
+    pyscf = _import_pyscf()
+
+    geometry = [
+        [symbol, tuple(np.array(coordinates)[3 * i : 3 * i + 3] * bohr_angs)]
+        for i, symbol in enumerate(symbols)
+    ]
+
+    mol = pyscf.gto.Mole()
+    mol.atom = geometry
+    mol.basis = basis
+    mol.charge = charge
+    mol.spin = mult - 1
+    mol.verbose = 0
+
+    mol_pyscf = mol.build()
+    if mult == 1:
+        molhf = pyscf.scf.RHF(mol_pyscf)
+    else:
+        molhf = pyscf.scf.ROHF(mol_pyscf)
+    _ = molhf.kernel()
+
+    core, active = qml.qchem.active_space(
+        mol.nelectron, mol.nao, mult, active_electrons, active_orbitals
+    )
+
+    one_ao = mol_pyscf.intor_symmetric("int1e_kin") + mol_pyscf.intor_symmetric("int1e_nuc")
+    two_ao = mol_pyscf.intor("int2e_sph")
+    one_mo = np.einsum("pi,pq,qj->ij", molhf.mo_coeff, one_ao, molhf.mo_coeff)
```

Comment:
Based on the benchmarking done, do you think [this](https://numpy.org/doc/stable/reference/generated/numpy.einsum.html) requires `optimize=True`?